06-building-a-program-start-to-finish/writing-html-js-css-outputs-(template)/output/index.html

```diff
@@ -0,0 +1,138 @@
+                    </tr>
+                    </tbody>
+
+                </table>
+            </div>
+            </div>
+            <div id="create-book-div" style="display:none;">
+                <p>create a new book here</p>
+                <br></br>
+                <label for="bookName">Name:</label>
+                <input id="bookName" type="text" name="bookName" placeholder = 'Enter the name of the file' ng-model="name">
+                <br></br>
+                <label for="bookTitle">Title:</label>
+                <input id="bookTitle" type="text" name="bookTitle" placeholder = 'Enter the title of the book' ng-model="title">
+                <br></br>
+                <label for="bookText">Text:</label>
+                <input id="bookText" type="text" placeholder="Enter a valid text/sentence, we would know you know! [20 words]" name="bookText" ng-model="text">
+                <br></br>
+                <p class="warning" id = "indexMsg"></p>
+                <br></br>
+                <a id = "link" style="display:none"></a>
+                <br></br>
+                <button class= "btn" id ="btnUpload" onclick = "switchDiv('upload-div', 'create-book-div')">Back</button>
+                <button class="btn" id="btnUpload" onclick="createBook()">done</button>
+            </div>
+
+            <div id="search-div" style="display:none">
+                <p>Select a file to search in or Search in all books</p>
+                <br></br>
+                <input id="id="search-text" type="text" placeholder="Enter word(s) to initiate search" ng-model = "query"/>
+                <button id="search-btn" class="btn" id="btnSearch" ng-click="search(query)">Search</button>
+                <button class= "btn" id ="btnUpload" onclick = "switchDiv('upload-div', 'search-div')">Back</button>
+                <div class="clear"></div>
+                <p ng-model="badMessage"></p>
+                <div id = "search-result-table">
+                <table>
+                    <thead>
+                    <tr>
+                        <th class="words" ></th>
+                        <th ng-repeat ="(mkey_1, mvalue_1) in library">{{mvalue_1}}</th>
+                    </tr>
+                    </thead>
+                    <tbody>
+                    <tr ng-repeat="(mkey_2, mvalue_2) in searchResult">
+                        <td>{{mkey_2}}</td>
+                        <td ng-repeat="(mkey, mval) in library">{{mvalue_2.indexOf(mval) !== -1 ? 'X':''}}</td>
+                    </tr>
+                    </tbody>
+
+                </table>
+            </div>
+
+
+                </table>
+            </div>
+            <div>
+                <p> {{badMessage}}</p>
+            </div>
+
+    </div>
+</div>
+<footer>
+  <p>By: Henshaw Rowland</p>
+</footer>
+<script type="text/javascript" src = "./inverted-index-app/InvertedIndex.js"></script>
```

Comment:
you can properly align your html tags and also avoid the ./ notation in browser environment. Instead you should use absolute or relative url's.